2020-06-01 18:31:03,124 [DEBUG] shapely.geos: Trying `CDLL(/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib)`
2020-06-01 18:31:03,133 [DEBUG] shapely.geos: Library path: '/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib'
2020-06-01 18:31:03,134 [DEBUG] shapely.geos: DLL: <CDLL '/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib', handle 7fb2f46a6e00 at 0x7fb32bfc1350>
2020-06-01 18:31:03,134 [DEBUG] shapely.geos: Trying `CDLL(/usr/lib/libc.dylib)`
2020-06-01 18:31:03,134 [DEBUG] shapely.geos: Library path: '/usr/lib/libc.dylib'
2020-06-01 18:31:03,135 [DEBUG] shapely.geos: DLL: <CDLL '/usr/lib/libc.dylib', handle 10f0786e0 at 0x7fb32bd4c4d0>
2020-06-01 18:51:40,551 [DEBUG] shapely.geos: Trying `CDLL(/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib)`
2020-06-01 18:51:40,560 [DEBUG] shapely.geos: Library path: '/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib'
2020-06-01 18:51:40,560 [DEBUG] shapely.geos: DLL: <CDLL '/Users/fzhu/Apps/miniconda3/envs/py3.7/bin/../lib/libgeos_c.dylib', handle 7f8a2bb2f4b0 at 0x7f8ad1196290>
2020-06-01 18:51:40,560 [DEBUG] shapely.geos: Trying `CDLL(/usr/lib/libc.dylib)`
2020-06-01 18:51:40,561 [DEBUG] shapely.geos: Library path: '/usr/lib/libc.dylib'
2020-06-01 18:51:40,561 [DEBUG] shapely.geos: DLL: <CDLL '/usr/lib/libc.dylib', handle 10a6bd6e0 at 0x7f8ac0af43d0>
